pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "glang"
+version = "0.1.0"
+authors = [
+  { name = "Antros Economides", email = "antroseconomides@hotmail.co.uk" },
+  { name = "Tom Grant", email = "thomas.grant.mail@gmail.com" },
+]
+description = "A Graphene front-end for LLVM"
+# TODO readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: GNU Affero General Public License v3",
+  "Operating System :: POSIX :: Linux",
+  "Programming Language :: Python :: 3",
+  "Topic :: Software Development :: Compilers",
+]
+dependencies = ["PyYAML==6.0.1", "typed-argument-parser==1.8.1"]
+
+[project.urls]
+"Homepage" = "https://github.com/DaveDuck321/GrapheneLang/"
+"Bug Tracker" = "https://github.com/DaveDuck321/GrapheneLang/issues"
+
+[project.scripts]
+glang = "glang.driver:main"
+
+[tool.hatch.envs.test]
+dependencies = ["interegular==0.3.2", "lark==1.1.7"]
+
+[tool.hatch.envs.test.scripts]
+run = "python tests/run_tests.py"
+
+[tool.hatch.envs.style]
+detached = true
+skip-install = true
+dependencies = ["black", "isort"]
+
+[tool.hatch.envs.style.scripts]
+check = [
+  "black --check --diff .",
+  "isort --check-only --diff --profile black .",
+]
+fmt = ["isort --profile black .", "black ."]
+
+[tool.hatch.build.targets.sdist]
+exclude = [".github", "docs", "tests"]
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/glang"]
```

Comment:
What pip actually installs. I still need to include the parser binary and the `lib` directory